server/src/main/java/org/spine3/server/entity/storage/Column.java

```diff
@@ -0,0 +1,235 @@
+    }
+
+    /**
+     * Creates new instance of the {@code Column} from the given getter method.
+     *
+     * @param getter the getter of the Column
+     * @param <T>    the type of the Column
+     * @return new instance of the {@code Column} reflecting the given property
+     */
+    public static <T> Column<T> from(Method getter) {
+        checkNotNull(getter);
+        final String name = nameFromGetterName(getter.getName());
+        final boolean nullable = getter.isAnnotationPresent(Nullable.class);
+        final Column<T> result = new Column<>(getter, name, nullable);
+
+        return result;
+    }
+
+    private static String nameFromGetterName(String getterName) {
+        final Matcher prefixMatcher = GETTER_PREFIX_PATTERN.matcher(getterName);
+        String result;
+        if (prefixMatcher.find()) {
+            result = prefixMatcher.replaceFirst("");
+        } else {
+            throw new IllegalArgumentException(
+                    format("Method %s is not a property getter", getterName));
+        }
+        result = Character.toLowerCase(result.charAt(0)) + result.substring(1);
+        return result;
+    }
+
+    /**
+     * Retrieves the name of the property which represents the Column.
+     *
+     * <p>For example, if the getter method has name "isArchivedOrDeleted", the returned value is
+     * "archivedOrDeleted".
+     *
+     * @return the name of the property reflected by this object
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Shows if the Column may return {@code null}s.
+     *
+     * @return {@code true} if the getter method is annotated as {@link Nullable},
+     * {@code false} otherwise
+     */
+    public boolean isNullable() {
+        return nullable;
+    }
+
+    /**
+     * Retrieves the Column value from the given {@link Entity}.
+     *
+     * @param source the {@link Entity} to get the Fields from
+     * @return the value of the Column represented by this instance of {@code Column}
+     */
+    public T getFor(Entity<?, ?> source) {
+        try {
+            @SuppressWarnings("unchecked")
+            final T result = (T) getter.invoke(source);
+            if (!nullable) {
+                checkNotNull(result, format("Not null getter %s returned null.", getter.getName()));
+            }
+            return result;
+        } catch (IllegalAccessException | InvocationTargetException e) {
+            throw new IllegalStateException(
+                    format("Could not invoke getter of property %s from object %s",
+                           getName(),
+                           source),
+                    e);
+        }
+    }
+
+    /**
+     * Retrieves the Column value from the given {@link Entity}.
+     *
+     * <p>The value is wrapped into a special container, which bears information about the Field's
+     * metadata.
+     *
+     * @param source the {@link Entity} to get the Fields from
```

Comment:
Please avoid capitalizing `Fields` everywhere. That's just a field.

You can refer to it as to an "Entity property" or "Entity field".